### ISSUE TITLE
Shouldn't it be log_sigmoid instead of softmax ?

### ISSUE THREAD

**Issue by karanchahal** [clcarwin/focal_loss_pytorch#7]
The paper mentions that the loss layer is combined with the sigmoid computation and not softmax. More speciafically this line 

```
Finally,
we note that the implementation of the loss layer combines
the sigmoid operation for computing p with the loss computation, resulting in greater numerical stability.
```

So isn't the author saying that we should use sigmoid activation over the last layer. The softmax usage maybe could lead to a lower accuracy.

**Comment by etienne87** [comment]
actually is there any benchmark for using focal softmax or binary cross-entropy?
